app/xeve_app.c

```diff
@@ -38,6 +38,8 @@
 #include <unistd.h>
 #endif
 
+<<<<<<< HEAD
+=======
```

Comment:
seems you have merge conflict markers, you may want to clear these up, usually compilers should show an error for these